### DISCUSSION TITLE
Starter guide/docs for gatsby + styled-components + ts

### DISCUSSION THREAD

**Discussion by alainkaiser** [ben-rogerson/twin.macro#625] [General]
Hi

Are there any plans for an additional starter guide / setup guide (docs) for the subsequent project stack:

- gatsby
- styled components
- TypeScript

I believe in the past there was documentation for this stack, but currently I can only find the JS docs, but no TS guide.

Thanks for the feedback.

**Comment by ben-rogerson** [comment]
Plans are there but I'm currently quite busy with a big update.

Try following the [non-ts gatsby instructions](https://github.com/ben-rogerson/twin.examples/tree/master/gatsby-styled-components) and coupling them with the [typescript instructions from the vite example](https://github.com/ben-rogerson/twin.examples/tree/master/vite-styled-components-typescript#add-typescript-types).
